core/alert/service_test.go

```diff
@@ -13,12 +13,12 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
-func TestService_Get(t *testing.T) {
+func TestService_List(t *testing.T) {
 	ctx := context.TODO()
 
-	t.Run("should call repository List method with proper arguments and return result in domain's type", func(t *testing.T) {
+	t.Run("should call repository Get method with proper arguments and return result in domain's type", func(t *testing.T) {
```

Comment:
Looks like repository `List` method is expected to be called.

Reply:
oh right, thanks @rohilsurana